src/Nethermind/Ethereum.Transition.Test/ByzantiumToConstantinopleTests.cs

```diff
@@ -33,7 +33,7 @@ public async Task Test(BlockchainTest test)
             await RunTest(test);
         }
         
-        public static IEnumerable<BlockchainTest> LoadTests() { var loader = new TestsSourceLoader(new LoadLegacyBlockchainTestsStrategy(), "bcByzantiumToConstantinople");
+        public static IEnumerable<BlockchainTest> LoadTests() { var loader = new TestsSourceLoader(new LoadBlockchainTestsStrategy(), "bcByzantiumToConstantinopleFix");
```

Comment:
same tests as above?

Reply:
My mistake. There is no ByzantiumToConstantinople test in ethereum tests repo anymore. I changed it to ByzantiumToConstantinopleFix which is already tested in another class. Will delete this file.